docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+# Autopilot Nginx demo
+#
+# For most use cases, it's better to build a custom image using this Nginx base image
+# see https://github.com/autopilotpattern/wordpress for an example
+
+nginx:
+    image: autopilotpattern/nginx
+    restart: always
+    mem_limit: 512m
+    env_file: _env
+    ports:
+        - 80
+    labels:
+        - triton.cns.services=nginx
+
+app:
+    image: autopilotpattern/containerbuddy-demo-app
```

Comment:
Okay, I've added a full example app to this repo. The app seems to work, I just need to integrate it with the rest of the repo.